### ISSUE TITLE
Poseconv3D (slowonly_r50_u48) reproducing results on diving48 dataset

### ISSUE THREAD

**Issue by a-github-learner** [open-mmlab/mmaction2#2025]
@kennymckormick   I try to reproduce Poseconv3D results on diving48 dataset under mmaction2 repository. However, I got around 20% accuracy after 200 epochs training on single GPU. How can i get similar accuracy under mmaction2 repository. Here's my config setting.
# model
> 
       model= dict(
          type='Recognizer3D',
          backbone=dict(
              type='ResNet3dSlowOnly',
              depth=50,
              pretrained=None,
              in_channels=17,
              base_channels=32,
              num_stages=3,
              out_indices=(2, ),
              stage_blocks=(4, 6, 3),
              conv1_stride_s=1,
              pool1_stride_s=1,
              inflate=(0, 1, 1),
              spatial_strides=(2, 2, 2),
              temporal_strides=(1, 1, 2),
              dilations=(1, 1, 1)),
          cls_head=dict(
              type='I3DHead', # I3DHead2
              in_channels=512,
              num_classes=48,
              spatial_type='avg',
              loss_cls=dict(type='CrossEntropyLoss'), # MultiSimilarity_CELoss
              dropout_ratio=0.5),
          train_cfg=dict(),
          test_cfg=dict(average_clips='prob'))
          dataset_type = 'PoseDataset'
          ann_file=tools/data/diving48/diving48_hrnet.pkl'
          left_kp = [1, 3, 5, 7, 9, 11, 13, 15]
          right_kp = [2, 4, 6, 8, 10, 12, 14, 16]
          train_pipeline = [
              dict(type='UniformSampleFrames', clip_len=48),
              dict(type='PoseDecode'),
              dict(type='PoseCompact', hw_ratio=1., allow_imgpad=True),
              dict(type='Resize', scale=(-1, 64)),
              dict(type='RandomResizedCrop', area_range=(0.56, 1.0)),
              dict(type='Resize', scale=(56, 56), keep_ratio=False),
              dict(type='Flip', flip_ratio=0.5, left_kp=left_kp, right_kp=right_kp),
              dict(
                  type='GeneratePoseTarget',
                  sigma=0.6,
                  use_score=True,
                  with_kp=True,
                  with_limb=False),
              dict(type='FormatShape', input_format='NCTHW'),
              dict(type='Collect', keys=['imgs', 'label'], meta_keys=[]),
              dict(type='ToTensor', keys=['imgs', 'label'])
          ]
          val_pipeline = [
              dict(type='UniformSampleFrames', clip_len=48, num_clips=1, test_mode=True),
              dict(type='PoseDecode'),
              dict(type='PoseCompact', hw_ratio=1., allow_imgpad=True),
              dict(type='Resize', scale=(-1, 64)),
              dict(type='CenterCrop', crop_size=64),
              dict(
                  type='GeneratePoseTarget',
                  sigma=0.6,
                  use_score=True,
                  with_kp=True,
                  with_limb=False),
              dict(type='FormatShape', input_format='NCTHW'),
              dict(type='Collect', keys=['imgs', 'label'], meta_keys=[]),
              dict(type='ToTensor', keys=['imgs'])
          ]
          test_pipeline = [
              dict(
                  type='UniformSampleFrames', clip_len=48, num_clips=10, test_mode=True),
              dict(type='PoseDecode'),
              dict(type='PoseCompact', hw_ratio=1., allow_imgpad=True),
              dict(type='Resize', scale=(-1, 64)),
              dict(type='CenterCrop', crop_size=64),
              dict(
                  type='GeneratePoseTarget',
                  sigma=0.6,
                  use_score=True,
                  with_kp=True,
                  with_limb=False, # False
                  double=True,
                  left_kp=left_kp,
                  right_kp=right_kp),
              dict(type='FormatShape', input_format='NCTHW'),
              dict(type='Collect', keys=['imgs', 'label'], meta_keys=[]),
              dict(type='ToTensor', keys=['imgs'])
          ]
          data = dict(
              videos_per_gpu=16,
              workers_per_gpu=2, 
              test_dataloader=dict(videos_per_gpu=1),
              train=dict(
                  type=dataset_type,
                  ann_file=ann_file_train,
                  split='train',
                  data_prefix='',
                  pipeline=train_pipeline),
              val=dict(
                  type=dataset_type,
                  ann_file=ann_file_val,
                  split='val',
                  data_prefix='',
                  pipeline=val_pipeline),
              test=dict(
                  type=dataset_type,
                  ann_file=ann_file_val,
                  split='val',
                  data_prefix='',
                  pipeline=test_pipeline))
  # optimizer
  

> 
     optimizer = dict(
          type='SGD', lr=0.4, #0.05
          momentum=0.9,
          weight_decay=0.0001) 
      optimizer_config = dict(grad_clip=dict(max_norm=40, norm_type=2))

# learning policy

>
      lr_config = dict(policy='CosineAnnealing', by_epoch=False, min_lr=0)
      total_epochs = 240
      checkpoint_config = dict(interval=1)
      workflow = [('train', 1),('val',1)] # [('train', 1),('val',1)]
      evaluation=dict(
          interval=1, 
          save_best='mean_class_accuracy', 
          metrics=['top_k_accuracy', 'mean_class_accuracy'],
          topk=(1, 5))
      log_config = dict(
          interval=20, hooks=[
              dict(type='TextLoggerHook'),
          ])
      dist_params = dict(backend='nccl')
      log_level = 'INFO'
      work_dir = './work_dirs/posec3d/slowonly_r50_u48_240e_diving48_keypoint_baseline'
      resume_from = None
      find_unused_parameters = False

**Comment by cir7** [comment]
hi, @a-github-learner , I guess that `lr` is too big for one single gpu, default `lr` in our config is used for 8 gpus, you may try scaling learning rate according to the total batchsize (num_gpus * videos_per_gpu)

**Comment by a-github-learner** [comment]
@cir7 Thanks for your timely reply! I would like to scale learning rate and retrain the module. The issued will be closed if it works. Thanks!
